src/main/java/com/yy5/employee/service/EmployeeService.java

```diff
@@ -36,5 +36,11 @@ public void update(int employeeNumber, String name, Integer age) throws Employee
                 .orElseThrow(() -> new EmployeeNotFoundException("EmployeeNumber " + employeeNumber + " is not found"));
         employeeMapper.update(employeeNumber, name, age);
     }
-}
 
+    public Employee delete(Integer employeeNumber) {
+        Employee employee = this.employeeMapper.findByEmployee(employeeNumber)
+                .orElseThrow(() -> new EmployeeNotFoundException("EmployeeNumber " + employeeNumber + " is not found"));
+        employeeMapper.delete(employeeNumber);
+        return employee;
+    }
+}
```

Comment:
削除後に、employeeは使っていないので、returnしないで良さそうです！
メソッドの返り値の型もvoidにしてあげると良いと思います。
```suggestion
    public void delete(Integer employeeNumber) {
        Employee employee = this.employeeMapper.findByEmployee(employeeNumber)
                .orElseThrow(() -> new EmployeeNotFoundException("EmployeeNumber " + employeeNumber + " is not found"));
        employeeMapper.delete(employeeNumber);
    }
}
```

Reply:
削除されたEntityを返す必要はないので`return employee`は削除しました。
deleteメソッドの戻り値が無いことを明示的にするため、戻り値の型を`Employee`から`void`に変更しました。
戻り値の型を変更したため、Controllerにおける戻り値の定義`Employee employee`も削除しました。
[b035f02](https://github.com/YYprogramer/employee/pull/15/commits/b035f023254cb7e85672b9bb84ea1124675dd513)